package.json

```diff
@@ -64,5 +64,8 @@
   },
   "engines": {
     "node": ">=v14.14.0"
+  },
+  "devDependencies": {
+    "file-lines-count": "^1.0.1"
```

Comment:
Since we don't have a distinction between development and deployment, all dependencies should just live in `"dependencies"`.